### PR TITLE
[ICONS] Clean up deprecated icons (#2147)

### PR DESCRIPTION
**Pull request by jeeyun**
Removing icons that were deprecated in 0.12. Leaving a couple that are slated for removal at 1.0.

Signed-off-by: Jeeyun Lim <jeeyun.lim@gmail.com>

### CODE REVIEW

File: src/clr-icons/shapes/technology-shapes.ts [hunk 1]
```diff
@@ -1168,7 +1163,6 @@ export const TechnologyShapes: any = {
   'process-on-vm': ClrShapeProcessOnVM,
 };
 
-Object.defineProperty(TechnologyShapes, 'analytics', descriptorConfig(TechnologyShapes['line-chart']));
```
Comment (Shijir) [inline src/clr-icons/shapes/technology-shapes.ts]: @jeeyun can you make sure we have this `"analytics"` alias in the `"chart-shapes"`? We need to remove the deprecated icons, but we shouldn't remove their aliases. The aliases should go into their new sets too.

Comment (jeeyun) [reply src/clr-icons/shapes/technology-shapes.ts]: Yeah I have already added it in this PR! :)

Comment (Shijir) [reply src/clr-icons/shapes/technology-shapes.ts]: oops I missed that sorry.

Comment (Shijir) [reply src/clr-icons/shapes/technology-shapes.ts]: And thanks again working on this :)

**Pull request merged**